BackofficeBundle/Form/Type/TinymceType.php

```diff
@@ -28,7 +28,7 @@ public function __construct(DataTransformerInterface $BBcodeTransformer)
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
-        $builder->addModelTransformer($this->bbCodeTransformer);
+//        $builder->addModelTransformer($this->bbCodeTransformer);
```

Comment:
I suggest to rename this form type into another thing than tinymce. Maybe wysibb ?